src/transport.ts

```diff
@@ -24,4 +45,23 @@ export class WebRTCTransport implements Transport {
 	get [symbol](): true {
 		return true
 	}
+
+	todo_cb() {
+	}
+
+	_connect (ma: Multiaddr, options: WebRTCDialOptions) {
+		this.channel = this.peer_connection.createDataChannel("channel");
+		this.channel.onopen = this.todo_cb;
+		this.channel.onclose = this.todo_cb;
+		this.peer_connection.onicecandidate = this.todo_cb;
+		this.peer_connection.setRemoteDescription(ma2sdp(ma));
```

Comment:
Refer https://github.com/libp2p/go-libp2p/pull/1655/files#diff-4f644a34f6d6d3f5ce6ddc58c2a2ff7e5dc3312006ec02810ef33a8016c529a1R52 for SDP construction.